# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1504029131 25200
#      Tue Aug 29 10:52:11 2017 -0700
# Node ID b95a09103ddc5ce29588af5e72baf39e0bacdc98
# Parent  df726a12c75778dc2c021f82056abad212fde12a
maint: fix typos of "daylight saving time"

* system.txi (Timing Utilities): Fix typo of "daylight saving time".
* oct-time.h (octave::base_tm): Likewise.
* now.m: Likewise.

diff --git a/doc/interpreter/system.txi b/doc/interpreter/system.txi
--- a/doc/interpreter/system.txi
+++ b/doc/interpreter/system.txi
@@ -73,17 +73,17 @@ Years since 1900.
 
 @item wday
 Days since Sunday (0-6).
 
 @item yday
 Days since January 1 (0-365).
 
 @item isdst
-Daylight Savings Time flag.
+Daylight saving time flag.
 
 @item gmtoff
 Seconds offset from UTC.
 
 @item zone
 Time zone.
 @end table
 
diff --git a/liboctave/system/oct-time.h b/liboctave/system/oct-time.h
--- a/liboctave/system/oct-time.h
+++ b/liboctave/system/oct-time.h
@@ -278,17 +278,17 @@ namespace octave
       int m_year;
 
       // Days since Sunday (0, 6).
       int m_wday;
 
       // Days since January 1 (0, 365).
       int m_yday;
 
-      // Daylight Savings Time flag.
+      // Daylight saving time flag.
       int m_isdst;
 
       // Time zone.
       long m_gmtoff;
 
       // Time zone.
       std::string m_zone;
 
diff --git a/scripts/time/now.m b/scripts/time/now.m
--- a/scripts/time/now.m
+++ b/scripts/time/now.m
@@ -42,17 +42,17 @@ function t = now ()
 
   ## The following doesn't work (e.g., one hour off on 2005-10-04):
   ##
   ##   seconds since 1970-1-1 corrected by seconds from GMT to local time
   ##   divided by 86400 sec/day plus day num for 1970-1-1
   ##   t = (time - mktime(gmtime(0)))/86400 + 719529;
   ##
   ## mktime (gmtime (0)) does indeed return the offset from Greenwich to the
-  ## local time zone, but we need to account for daylight savings time
+  ## local time zone, but we need to account for daylight saving time
   ## changing by an hour the offset from CUT for part of the year.
 
 endfunction
 
 
 %!assert (isnumeric (now ()))
 %!assert (now () > 0)
 %!assert (now () <= now ())
